docs/root/intro/arch_overview/security/threat_model.rst

```diff
@@ -15,16 +15,31 @@ highest priority concerns. Availability, in particular in areas relating to DoS
 exhaustion, is also a serious security concern for Envoy operators, in particular those utilizing
 Envoy in edge deployments.
 
-The Envoy availability stance around CPU and memory DoS, as well as Query-of-Death (QoD), is still
-evolving. We will continue to iterate and fix well known resource issues in the open, e.g. overload
-manager and watermark improvements. We will activate the security process for disclosures that
-appear to present a risk profile that is significantly greater than the current Envoy availability
-hardening status quo. Examples of disclosures that would elicit this response:
-
-* QoD; where a single query from a client can bring down an Envoy server.
```

Comment:
I notice that the term QoD and definition have disappeared. Was this deliberate?

Reply:
Yes, I think more general "availability issue" covers it. I can add more explicit call out for it.

Reply:
Yeah, maybe just call it out, as it's a common case. Thanks.

Reply:
done